internal/exec/stages/fetch_offline/fetch_offline_test.go

```diff
@@ -64,6 +64,8 @@ func TestConfigNeedsNet(t *testing.T) {
 		},
 	}
 	assertNotNeedsNet(t, &cfg)
+	cfg.Storage.Files[0].Contents.Source = util.StrToPtr("")
```

Comment:
Can we also add a test for `nil` here? (I.e. not specified). This tests `sourceNeedsNet`.

Reply:
Done.